src/types/swingset.ts

```diff
@@ -35,6 +35,8 @@ export type SwingSetParams = {
   bootstrap_vat_config: string;
   power_flag_fees: PowerFlagFee[];
   queue_max: QueueMax[];
+  bundle_uncompressed_size_limit_bytes: string;
+  chunk_size_limit_bytes: string;
```

Comment:
Do these need to accomodate SwingSet versions that lack bundle chunking params?

```suggestion
  bundle_uncompressed_size_limit_bytes?: string;
  chunk_size_limit_bytes?: string;
```